spec/model/post_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'associations' do
+    it 'belongs to a user' do
+      user = User.create(name: 'John Doe', photo: 'https://example.com/photo.jpg', bio: 'User bio')
+      post = Post.new(title: 'Hello', text: 'This is my post')
+      post.user = user
+      post.save
+
+      expect(post.user).to eq(user)
+    end
+  end
+end
```

Comment:
- [ ] I hope everything is going well with you, thank you for letting me know that you are doing your best despite the hospitalization. About the milestone the objective is to create tests for each model, so you were doing it well before the last review, now the test is passing without testing all the validations and methods from the models I mean as you have done with the post model (review the image below) you should complete the tests for each validation and methods in the models as required for this milestone 👇🏽 

![image](https://github.com/mercymugambi/blog-app/assets/58642949/e206c818-80c2-4277-8462-601e056ae8ba)

![image](https://github.com/mercymugambi/blog-app/assets/58642949/98ad382f-6ea8-4e78-8e43-2b92e071100a)